src/auth-service/controllers/create-user.js

```diff
@@ -220,7 +220,18 @@ const createUser = {
   },
 
   verify: (req, res) => {
-    return res.status(httpStatus.OK).send("this token is valid");
+    logText("..................................");
+    logText("user verify......");
```

Comment:
@BenjaminSsempala , the other error we are getting via Slack is here but it is a bit hard to replicate or even understand fully! :) 

So the purpose of this PR just changed to refactoring since I could not fix this.